server/weaverbird/steps/append.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+
+from pandas import DataFrame
+from pydantic import Field
+
+from weaverbird.steps.base import BaseStep
+from weaverbird.types import PipelineExecutor
+
+Pipeline = List[dict]
+
+
+class AppendStep(BaseStep):
+    name = Field('append', const=True)
+    pipelines: List[Pipeline]
+
+    def execute(
+        self,
+        df: DataFrame,
+        domain_retriever,
+        execute_pipeline: PipelineExecutor,
+    ) -> DataFrame:
+        for pipeline in self.pipelines:
+            other_df = execute_pipeline(pipeline)
+            df = df.append(other_df, ignore_index=True)
```

Comment:
You can directly append a list of dataframes:

```python
In [1]: import pandas as pd

In [2]: df = pd.DataFrame([[1, 2], [3, 4]], columns=list('AB'))

In [3]: df2 = pd.DataFrame([[5, 6], [7, 8]], columns=list('AB'))

In [4]: df3 = pd.DataFrame([[9, 10], [11, 12]], columns=list('AB'))

In [5]: df.append([df2, df3])
Out[5]: 
    A   B
0   1   2
1   3   4
0   5   6
1   7   8
0   9  10
1  11  12
```

Reply:
feacdde